client/tests/integration/triggers/time_trigger.rs

```diff
@@ -69,18 +68,14 @@ fn time_trigger_execution_count_error_should_be_less_than_10_percent() -> Result
 }
 
 /// Build register ISI for trigger which mints roses
-fn build_register_trigger_isi(
-    account_id: AccountId,
-    asset_id: AssetId,
-    schedule: TimeSchedule,
-) -> Result<RegisterBox> {
-    let instruction = MintBox::new(1_u32, asset_id);
+fn build_register_trigger_isi(asset_id: AssetId, schedule: TimeSchedule) -> Result<RegisterBox> {
```

Comment:
I'd like this function to be part of our public API. Most of our downstream clients may want something like this.

Reply:
Created #2009